java/src/main/java/ai/rapids/cudf/CuFileBuffer.java

```diff
@@ -30,16 +31,31 @@
    * @param buffer         The device memory buffer used for the cuFile buffer.
```

Comment:
The Javadoc here should mention the ownership semantics and how this buffer will be closed when the `CuFileBuffer` instance is closed.

Reply:
Done.